packages/amplify_datastore/example/integration_test/clear_test.dart

```diff
@@ -22,7 +22,7 @@ void main() {
       await Amplify.DataStore.save(blog);
       var resultOne = await Amplify.DataStore.query(Blog.classType);
       expect(resultOne, isNotEmpty);
-      await Amplify.DataStore.clear();
+      await clearDataStore();
```

Comment:
`clearDataStore();` adds a delay before calling clear, which is still needed since this test calls clear and then start multiple times in a row.